sdk/src/main/java/io/opentelemetry/sdk/trace/MultiSpanProcessor.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+final class MultiSpanProcessor implements SpanProcessor {
+  private final List<SpanProcessor> spanProcessors;
+
+  static SpanProcessor create(List<SpanProcessor> spanProcessors) {
+    return new MultiSpanProcessor(Collections.unmodifiableList(new ArrayList<>(spanProcessors)));
+  }
+
+  @Override
+  public void onStartSync(ReadableSpan readableSpan) {
```

Comment:
Nit: `checkNotNull`? Same for `onEndSync`.

Reply:
I wouldn't do this because this is not a public API (I mean it is called only by the tracer). Unnecessary if statement (jump).